dataactbroker/handlers/fileGenerationHandler.py

```diff
@@ -123,8 +82,3 @@ def generate_e_file(task, submission_id, job_id, timestamped_name, upload_file_n
         session.commit()
 
         write_csv(timestamped_name, upload_file_name, is_local, fileE.Row._fields, rows)
-
-
-if __name__ in ['__main__', 'jobQueue']:
-    configure_logging()
-    celery_app.conf.update(BROKER_URL=broker_url('localhost'))
```

Comment:
Do we still have to configure_logging if it's `__main__` or is this never called individually so logging is already configured?